### DISCUSSION TITLE
BT curve not showing on graph

### DISCUSSION THREAD

**Discussion by ErezMinuto** [artisan-roaster-scope/artisan#1367] [Q&A]
[artisan-settings.txt](https://github.com/artisan-roaster-scope/artisan/files/13785408/artisan-settings.txt)
![btnotshown](https://github.com/artisan-roaster-scope/artisan/assets/152774771/2e537733-cfe4-400d-be24-bfc5b556a357)

![settings](https://github.com/artisan-roaster-scope/artisan/assets/152774771/0864fa32-47c0-40df-87b6-20fac60a450f)
version Artisan 2.10.0 (c44a180) 
Everything was ok and a few days ago the curve of the BT temperature stopped appearing.
also attached setting file

**Comment by poundy** [comment]
your settings file shows you have the BT curve turned off, with BTCurve=false


ETcurve=true
BTcurve=false
ETlcd=true
BTlcd=true

I'm sure you've pressed a key to turn the BT curve off. Perhaps it's the "i" key, check out the keyboard shortcuts in the help menu.

**Comment by ErezMinuto** [comment]
Checked the keyboard shortcuts and it says "i" is the key but nothing happens when i press it.
I pressed it while roasting, should i wait till roast ends?

**Reply by Terracotta-6** [reply]
I have the same problem Maybe it's a compatibility issue with the old profile , upload Profiles.alog .

**Reply by ErezMinuto** [reply]
alog file of the roast?

**Reply by Terracotta-6** [reply]
yes

**Reply by ErezMinuto** [reply]
[CTE Silon Touch  2023-12-28.txt](https://github.com/artisan-roaster-scope/artisan/files/13786611/CTE.Silon.Touch.2023-12-28.txt)

**Comment by Terracotta-6** [comment]
![5fff007e80fa041f62688f9e440feb2](https://github.com/artisan-roaster-scope/artisan/assets/149906183/5fe9f036-a46d-4e1e-bf2e-ad987368b71d)

**Reply by ErezMinuto** [reply]
is it the latest version?

**Reply by Terracotta-6** [reply]
Continuous build

**Comment by Terracotta-6** [comment]
![feb219e55892583f4eeaeb2016464c6](https://github.com/artisan-roaster-scope/artisan/assets/149906183/039bc737-1656-446e-8afc-627eb9fc232e)

**Comment by ErezMinuto** [comment]
ok, found the solution just click on the bt temp on the right side of the screen and the curve will apear again

**Reply by Terracotta-6** [reply]
Made a silly mistake.
![0f2dce4385874ccb9f93ceeb651e53e](https://github.com/artisan-roaster-scope/artisan/assets/149906183/f3ec36ac-ca98-42ea-8498-5e3a101a81d9)
